yarn.lock

```diff
@@ -10177,9 +10177,9 @@ verror@1.3.6:
   dependencies:
     extsprintf "1.0.2"
 
-"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#2b1e3463725279589728e666f478a756769bf580":
-  version "3.76.0"
-  resolved "https://github.com/department-of-veterans-affairs/vets-json-schema.git#2b1e3463725279589728e666f478a756769bf580"
+"vets-json-schema@https://github.com/department-of-veterans-affairs/vets-json-schema.git#f42bad3ddff042d2700dd93a2dccd646a755f905":
+  version "3.79.0"
```

Comment:
This is a couple versions up, it looks like. Are the other changes in between just to the 526, or other forms?

Reply:
Fair question! These were all small 526-related changes that didn't have any impact on the form / config.
In any case, looks as though someone bumped the version to 3.80 in the meantime 😬